acm/odh-edge/apps/telemetry/otel-collector.yaml

```
@@ -0,0 +1,33 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: OpenTelemetryCollector
+metadata:
+  name: cluster-collector
+  namespace: ai-edge-collector
```

I know kustomize would overwrite this, but it would be good to either omit it or set it to
```suggestion
  namespace: opendatahub-edge-collector
```
so it's unified.

---

:+1: